src/coreclr/vm/peimagelayout.cpp

```diff
@@ -907,9 +907,10 @@ static bool HavePlaceholderAPI()
         return false;
     }
 
+#ifdef TARGET_WINDOWS
```

Comment:
This is not needed, this whole function plus some other stuff is under this ifdef already. See line 874